pom.xml

```diff
@@ -72,12 +81,56 @@
       <artifactId>vertx-rx-java</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.termd</groupId>
+      <artifactId>termd-core</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
+    </dependency>
 
-    <!-- Not needed at runtime unless you use the JUnit extension -->
+    <!-- -->
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.12</version>
+      <groupId>io.vertx</groupId>
```

Comment:
yes it does depend on this for now for reporting metrics, there are a couple of commands. I think for now it's fine as this is a "preview" shell.

in 3.2 we should do the reverse, have dropwizard depends on Vert.x Shell and provide commands in the metrics project and not in Vert.x Shell.